docker-compose.yml

```diff
@@ -1175,6 +1177,25 @@ services:
       CHECK_WHEEL_CONTENT: "ON"
     command: /arrow/ci/scripts/python_wheel_unix_test.sh /arrow
 
+  # TODO: Remove this when the official Docker Python image supports the free-threaded build
```

Comment:
Is there an upstream ticket we can link to here?

Reply:
Added.